lib/base64.hoon

```diff
@@ -5,35 +5,96 @@
 ::
 =+  [pad=& url=|]
 |%
+::
++$  byte    @D
++$  word24  @
+::
+++  div-ceil
+  ::  divide, rounding up.
+  |=  [x=@ y=@]  ^-  @
+  ?:  =(0 (mod x y))
+    (div x y)
+  +((div x y))
+::
+++  explode-bytes
+  ::  Explode a bytestring into list of bytes. Result is in LSB order.
+  |=  =octs  ^-  (list byte)
+  =/  atom-byte-width  (met 3 q.octs)
+  =/  leading-zeros    (sub p.octs atom-byte-width)
+  (weld (reap leading-zeros 0) (rip 3 q.octs))
+::
+++  explode-words
+  ::  Explode a bytestring to words of bit-width `wid`. Result is in LSW order.
+  |=  [wid=@ =octs]
+  ^-  (list @)
+  ~&  %explode-words
```

Comment:
You left sigpams in the final code.

Reply:
Yes.

Reply:
I'll remove them after review; still useful for now.